### PR TITLE
Im 128 log service

### PR DESCRIPTION
**Pull request by NamespaceValentine**
You can use the createUser route to test this.

### CODE REVIEW

**Comment by NamespaceValentine** [comment]
This is ready for review.

**Comment by TimSwart** [comment]
Getting error when creating a user:
Error: TimeoutError: Missing credentials in config

File: routes/Login/createUser.js [hunk 1]
```diff
@@ -66,6 +56,28 @@ router.route("/").post(function(req, res) {
...
+        .then(function() {
+            require('../../imp_services/implogging')(req.cookies.IMPId, function(logService){
+                logService.action.value = username;
+                logService.setType(800);
+                logService.store(function(err, results){
+                    if(err){
+                        res.status(500).send(err);
+                    }
+                    else{
+                        SendConfirmation(email, function (err) {
```
Comment (TimSwart) [inline routes/Login/createUser.js:69]: does the SendConfirmation() function have to be inside of logService.store(), or can it follow after that in its own .then() promise?

Comment (NamespaceValentine) [inline routes/Login/createUser.js:69]: I will be having John help me turn this into a promise chain later. I just am not as good at promises as he is.

Comment (NamespaceValentine) [inline routes/Login/createUser.js:69]: Added a todo

Comment (TimSwart) [inline routes/Login/createUser.js:69]: cool. I think you're good to merge then.

Comment (TimSwart) [inline routes/Login/createUser.js:69]: my thought was just that everything else in that function has to do with logging, so maybe there should be a distinction between that and the email confirmation

Comment (NamespaceValentine) [inline routes/Login/createUser.js:69]: Thats the point here, the SendConfirmation is handled in its own function, but shouldnt be sent until after the store happens.

**Pull request merged**